mobile/build.gradle

```diff
@@ -17,7 +17,7 @@ buildscript {
 }
 
 apply plugin: 'com.android.application'
-if(!isFoss) {
+if (!isFoss) {
```

Comment:
The formatting changes here and in the manifest were done by Android Studio when I added the new activity.